Trappist/src/Promact.Trappist.Core/Controllers/QuestionController.cs

```diff
@@ -111,6 +111,23 @@ public QuestionController(IQuestionRepository questionsRepository, UserManager<A
             }
             return Ok(questionAC);
         }
+
+        /// <summary>
+        /// Returns Question of specific Id
+        /// </summary>
+        /// <param name="id">Id of Question</param>
+        /// <returns>
+        /// QuestionAC class object
+        /// </returns>
+        [HttpGet("{id}")]
+        public async Task<IActionResult> GetQuestionByIdAsync([FromRoute]int id)
+        {
+            if (id <= 0)
```

Comment:
What if i pass 10000 in id and there are only 100 questions in database. What it will return?